### ISSUE TITLE
Parallelization Bug: reports for batches with spaces in the testable name are not collated properly

### ISSUE THREAD

**Issue by lyndsey-ferguson** [lyndsey-ferguson/fastlane-plugin-test_center#109]
### New Issue Checklist

- [x] Updated `fastlane-plugin-test_center` to the latest version
- [x] I read the [README.md](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md)
- [x] I reviewed the [example(s)](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md) for the action(s) I am using
- [x] I have removed any sensitive data such as passwords, authentication tokens, or anything else I do not want to world to see

When a test target has a space in it, the directory where reports are written to have the spaces escaped in the name. When the entire run of all batches is complete, the report files are not collated because multi_scan is looking for them in testable directories without the escaped spaces.

For example:
```
- test_output
  - Atomic\ Dog\ Tests-batch-1
    - report.xml
  - Atomic\ Dog\ Tests-batch-2
     - report.xml
```

The batched reports should be written to `Atomic Dog Tests-batch-1` in the above example.

##### Complete output when running fastlane, including the stack trace and command used

<details>
  <pre>[INSERT OUTPUT HERE]</pre>
</details>

### Environment

<details>
  <pre>[INSERT OUTPUT HERE]</pre>
</details>

**Comment by jineshqa** [comment]
Hi @lyndsey-ferguson Could you also post an example for using parallel tests.

**Comment by jineshqa** [comment]
is this the way?

parallel_testrun_count: 4

**Comment by lyndsey-ferguson** [comment]
Yes, @jineshqa. Please see the [pull request](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/pull/104). I have an example there.

**Comment by lyndsey-ferguson** [comment]
Fixed in [v3.8.0.parallelizing.beta.5](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/releases/tag/v3.8.0.parallelizing.beta.5)
